_includes/authorizations-resource.md

```diff
@@ -0,0 +1,191 @@
+{% assign payment-instrument = include.payment-instrument | default: creditcard %}
+{% assign payment-id = "77DA8F11-43DF-4E59-800A-EA0E94D27F4A" %}
+{% assign transaction-id = "373B335B-C0EE-46B2-A123-8A2B6C9DA2EA" %}
+
+### Authorizations
+
+The `authorizations` resource will list the authorization transactions
+made on a specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/{{payment-instrument}}/payments/{{payment-id}}/authorizations HTTP/1.1
+Host: api.externalintegration.payex.com
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": "/psp/{{payment-instrument}}/payments/{{payment-id}}",
```

Comment:
Looks like this code example is indented with 2 spaces instead of 4. Please ensure that all code examples are indented with 4 spaces.

Reply:
Most of everything I write and submit are indented with 2 spaces instead of 4.